### ISSUE TITLE
Image component should support raw stream SVG

### ISSUE THREAD

**Issue by chrisglein** [microsoft/react-native-windows#3917]
Forked out from #3573. More info there.

> #3845 adds basic svg support based on file extension. Raw stream is not supported. We also don't currently have a good way to get the width and height off of the svg file, so developers will need to specify them in the Image's style properties.
> https://docs.microsoft.com/en-us/windows/win32/direct2d/svg-support

We've implemented SVG support based on file extension. This issue tracks adding it based on raw stream. Partners are unblocked with what we have, so this remaining work is lower priority.

**Comment by marlenecota** [comment]
Closing. This was implemented in #7198
